localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -0,0 +1,35 @@
+﻿l_english:
+ ###REGIONAL###
+ ##############
+ # Tribal Era #
+ ##############
+
+ innovation_elephantry:0 "War Elekks"
+ innovation_elephantry_desc:0 "Elekks can be tamed and used to strike our foes on the battlefield.  They tower over any foe and can punch through any enemy line."
+ 
+ innovation_war_camels:0 "War Camels"
+ innovation_war_camels_desc:0 "To traverse the sands of Azeroth our people need a trusty steed to carry us. The scorching would have made any horse would have give up by now, but camels are not phased byt the sun at all.  That type of rugedness can be relied on in a battle."
```

Comment:
Doesn't work.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104960906-b218f280-59ee-11eb-966c-97467a109e32.png)

</details>